contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "In-depth: PostHog vs Fathom"
+date: 2024-11-11
```

Comment:
If you set the date in the future, we can't see the article in the preview. You can change it back once you publish.
```suggestion
date: 2024-11-08
```